pkg/config/config.go

```diff
@@ -20,14 +20,23 @@ import (
 )
 
 const (
-	localConfigEnvName    = "LOCALODOCONFIG"
 	configFileName        = "config.yaml"
 	localConfigKind       = "LocalConfig"
 	localConfigAPIVersion = "odo.openshift.io/v1alpha1"
+
 	// DefaultDebugPort is the default port used for debugging on remote pod
 	DefaultDebugPort = 5858
 )
 
+// Env variables
+const (
+	// Local config
+	localConfigEnvName = "LOCALODOCONFIG"
+
+	// Setting this env to true will expose experimental features to the user
+	OdoExperimentalEnv = "ODO_EXPERIMENTAL"
```

Comment:
as this environment variable is related to a preference it should be moved to the `preference` package

Reply:
Wouldn't it be a good idea, to have all your env variables at the same place in the `config` package, over scattering env variables around different packages?

Reply:
But this env variable isn’t related to config. Its related to a preference so in my opinion it should be in preferences

Reply:
And we currently don’t have all env variables in config e.g. https://github.com/openshift/odo/blob/abbf342023f23b364131eb59c0677d942a5dde36/pkg/preference/preference.go#L19
so I would definitely prefer env variables being more contextual then having them in the same place.

Reply:
Done.
But, I strongly feel it is better idea to have all env variables in a single package than having it scattered in all the packages.